Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
```

Comment:
That's where things start to get confusing. 440900118, 440900118_2 and 440900119 are not even close to what's being said.

I'll take a look at the script files, most likely the PS3 text is different.